privacyscore/backend/management/commands/scanfromfile.py

```diff
@@ -51,10 +41,14 @@ def handle(self, *args, **options):
                 if '.' in url:
                     url = normalize_url(url)
                     site = Site.objects.get_or_create(url=url)[0]
-                    if scan_list:
-                        site.scan_lists.add(scan_list)
                     sites.append(site)
 
+        if options['create_list_name']:
```

Comment:
Pull the data out of the object only once, e.g. `list_name = options['create_list_name']`. If we ever want to modify what's coming out of the object, we'll appreciate that there'll be only one place to touch.